org.eclipse.scanning.event/src/org/eclipse/scanning/event/ConsumerImpl.java

```diff
@@ -620,7 +593,7 @@ public void pause() throws EventException {
 			awaitPaused = true;
 			if (mconsumer!=null) mconsumer.close();
 			mconsumer = null; // Force unpaused consumers to make a new connection.
-			logger.info(getName()+" is paused");
+			LOGGER.info(getName()+" is paused");
 			System.out.println(getName()+" is paused");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)